agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -298,15 +297,14 @@ func (cs *CredentialSpecResource) GetName() string {
 // Create is used to create all the credentialspec resources for a given task
 func (cs *CredentialSpecResource) Create() error {
 	var err error
-	// To fail fast, check execution role first
+	var iamCredentials credentials.IAMRoleCredentials
+
 	executionCredentials, ok := cs.credentialsManager.GetTaskCredentials(cs.getExecutionCredentialsID())
-	if !ok {
-		// No need to log here. managedTask.applyResourceState already does that
-		err = errors.New("credentialspec resource: unable to find execution role credentials")
-		cs.setTerminalReason(err.Error())
-		return err
+	if ok {
+		iamCredentials = executionCredentials.GetIAMRoleCredentials()
+	} else {
+		seelog.Debugf("Unable to obtain execution credentials: %v", ok)
```

Comment:
This change will make it not return error when cannot find execution role, you should not change this.

Reply:
We do not need the credentials for using the file option. This should only be required for the other options. We also want to fail later in this situation.